src/olympia/abuse/tests/test_cinder.py

```diff
@@ -2456,7 +2456,11 @@ def test_report_additional_context_error(self):
 
 class TestCinderRating(BaseTestCinderCase, TestCase):
     CinderClass = CinderRating
-    expected_queries_for_report = 1  # For the author
+    expected_queries_for_report = 4
```

Comment:
fwiw, I think the testing is a little deficient around queries - the `CinderAddon` tests only report _2_ queries, while exposing some of the add-on metadata in `CinderRating` added **3** additional queries.  My suspicion is the reloading of the `AbuseReport` to clear the preloaded db data isn't clearing all caches - I poked around with it but couldn't get any of the existing tests to break so 🤷‍♂️ .